### DISCUSSION TITLE
Master Updated!

### DISCUSSION THREAD

**Discussion by Curtis2012** [Curtis2012/dewheater#10] [Announcements]
I just merged new features into the master branch. These include:

- support of either DHT22 or BME280 sensors
- Maximum Dew Heater Time On parameter. If set, limits the number of fractional hours that the dew heater can be "on"
- Expanded status output
- A few other minor cleanups

Testers welcome!!

**Comment by clint70** [comment]
NICE!... I will update this afternoon.

**Comment by Curtis2012** [comment]
Pushed another minor change to master just now.

- added handling for SIGTERM signal. This signal is received when a service is stopped by systemctl. Handling this signal allows dewheater to cleanup before it exits. It now executes a GPIO.cleanup() routine so that GPIO resources as properly released.

**Reply by Dentonknifeworks** [reply]
I too will have a look later this week. Been running the last one with no issues. Well, I did have a BME280 die on me LOL but not the fault of the code.

**Reply by Curtis2012** [reply]
A good way to get familiar with the new features is just to go thru the README file that describes each configuration parameter.

**Comment by Curtis2012** [comment]
Fixed a bug today that was causing dewHeater object to be instantiated twice and the config file to be processed and displayed twice. 

Added forced shut off of dew heater relay to SIGTERM handler. This avoids the dew heater being left "on" after stopping the service.

These changes were promoted to master. Update your code base to get them.
